src/runtime/less/ts-modals.less

```diff
@@ -4,6 +4,21 @@
 	border-radius: 0;
 	width: 100%;
 	height: 100%;
+	&.ts-micro {
+		max-width: 769px;
```

Comment:
It is weird to use the numbers here. But I don't know how to fix these.

Reply:
Yeah! Also for the `100px` margin!
I just putted the same width from the design! which I think is bootstrap 768px + 1px breakpoint for mobile!

Reply:
But the mobile breakpoint for the `ts-ui` is `650` as I found out! So, not sure what to do!